modules/mesh-task/validation.tf

```diff
@@ -3,11 +3,9 @@
 
 locals {
   require_port_unless_no_port_true                          = !var.outbound_only && var.port == 0 ? file("ERROR: port must be set if outbound_only is false") : null
-  require_ca_cert_if_tls_enabled                            = (var.tls && var.consul_server_ca_cert_arn == "") ? file("ERROR: consul_server_ca_cert_arn must be set if tls is true") : null
```

Comment:
Removed CA cert related validation because we now have multiple ways to configure ca certs (Via Env vars, via files or protocol specific TLS settings which makes validation a lot more complex)